src/components/speaking-event/speaking-event.stories.mdx

```diff
@@ -0,0 +1,37 @@
+import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+import template from './speaking-event.twig';
+
+<Meta title="Components/Speaking Event" />
```

Comment:
Honest question: What about this component is specific to a _speaking_ event? If there was a charity event, would it need a different component? Just pushing a little on the "speaking" adjective in case it isn't necessary. 🙂

Reply:
I appreciate this quesition. I'm open to a more general name! I also was bothered by it being so specific but had trouble thinking of an alternative.

Reply:
Would "Event" be too general? 🤔

Reply:
> Would "Event" be too general? 🤔

As in, I'd like to propose `Event` as the component name (which now re-reading your first comment I think you are suggesting this as well).

Reply:
Or it could be "event summary" or "event tease" or something that's more about the purpose of the component and less about the component's content?

Reply:
@tylersticka Thanks, great suggestions! I'll use one of those. 😉

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1488/commits/a16711c6680c16c6eabdb1ed8fa5caad2b1597c0